io/FV3GFS_io.F90

```diff
@@ -1072,6 +1076,8 @@ subroutine sfc_prop_restart_read (Sfcprop, Atm_block, Model, fv_domain)
     ! TODO: move to physics and stop building namelist_soilveg/set_soilveg
     ! in the FV3/non-CCPP physics when the CCPP-enabled executable is built.
 #endif
+#if 1
```

Comment:
This is not really needed. It's more a reminder that we can remove this once the Noah-MP initialization/calculation of variables in this file is moved to CCPP. We do have the code ready inside CCPP, but the problem is that the calculation in FV3GFS_io.F90 follows the precision of the FV3 code outside of gfsphysics, i.e. it is 32bit in case the code is built with 32BIT=Y, while the calculation in CCPP is always in the physics precision (64bit). When the IPD is phased out, we need to agree on doing this calculation in CCPP in 64bit (or whatever the precision for the physics is). Long story short, I will remove those #if 1 and corresponding #endif statements.